src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('.tree li');
+
+listItems.forEach((item) => {
+  if (item.querySelector('ul')) {
+    const span = document.createElement('span');
+    const originalText = item.firstChild.textContent.trim();
+
+    span.textContent = originalText;
+    item.firstChild.textContent = '';
+    item.insertBefore(span, item.firstChild);
+  }
+});
+
+const titles = document.querySelectorAll('span');
+
+titles.forEach((title) => {
+  title.addEventListener('click', () => {
+    if (title.nextElementSibling.style.display === 'none') {
```

Comment:
Consider setting the initial display state of the nested lists to 'none' to ensure that the toggle functionality works correctly on the first click. This can be done by iterating over the nested lists and setting their display style to 'none' when the page loads.